naiserator-dev.yaml

```
@@ -46,11 +46,15 @@ spec:
         mountPath: /secrets/eia/credentials
       - kvPath: oracle/data/dev/config/eia2_q1
         mountPath: /secrets/eia/config
-      - kvPath: /kv/preprod/fss/syfoservice-data-syfosmregister/default
+      - kvPath: /kv/preprod/fss/syfoservice-data-syfosmregister/teamsykmelding
         mountPath: /var/run/secrets/nais.io/vault
+  azure:
+    application:
+      enabled: true
   webproxy: true
   ingresses:
     - "https://syfoservice-data-syfosmregister.nais.preprod.local"
+    - "https://macgyver.nais.dev.intern.nav.no"
```

```suggestion
    - "https://macgyver.dev.intern.nav.no"
```
https://doc.nais.io/clusters/on-premises/#dev-fss

---

aha, takk :D